lib/rules/amd-function-arity.js

```diff
@@ -30,10 +34,35 @@ module.exports = function (context) {
         return node.type === "FunctionExpression";
     }
 
+    function findAllowedExtraDependencies() {
```

Comment:
From here on out, you just need to check if it's an array and act accordingly, see comment below.